scripts/sync-dist.sh

```diff
@@ -8,21 +8,21 @@ UHC_PORTAL=${UHC_PORTAL:-../uhc-portal}
 
 function sync {
   if [ -d "${FACET_ROOT}" ]; then
-    rsync -avz --delete ${BUILD_OUTPUT} ${FACET_ROOT}/node_modules/facet-lib
+    rsync -avz --delete ${BUILD_OUTPUT} ${FACET_ROOT}/node_modules/assisted-ui-lib
   else
-    echo Sync to ${FACET_ROOT}/node_modules/facet-lib skipped
+    echo Sync to ${FACET_ROOT}/node_modules/openshift-assisted-ui-lib skipped
   fi
 
   if [ -d "${UHC_PORTAL}" ]; then
-    rsync -avz --delete ${BUILD_OUTPUT} ${UHC_PORTAL}/node_modules/facet-lib
+    rsync -avz --delete ${BUILD_OUTPUT} ${UHC_PORTAL}/node_modules/assisted-ui
```

Comment:
this should be `openshift-assisted-ui-lib` ?

Reply:
good catch, not urgent, will do in a follow-up